web/auth/auth.go

```diff
@@ -231,12 +291,22 @@ func IsLoggedIn(c echo.Context) bool {
 
 // Routes sets the routing for the status service
 func Routes(router *echo.Group) {
+	noCSRF := middleware.CSRFWithConfig(middleware.CSRFConfig{
+		TokenLookup:    "form:csrf_token",
+		CookieMaxAge:   3600, // 1 hour
+		CookieHTTPOnly: true,
+		CookieSecure:   true,
+	})
```

Comment:
could it be interesting to store the CSRF in the session instead of using another cookie ?